src/main/java/Fundamentals/oop/BankAccount.java

```diff
@@ -0,0 +1,63 @@
+package Fundamentals.oop;
+
+public class BankAccount {
+    private String type;
+    private double balance;
+    private String currency;
+    private String accountType;
+
+    public BankAccount(String type, String currency) {
+        this.type = type;
+        this.currency = currency;
+        this.balance = 0;
+    }
+
+    public BankAccount() {
+        this.type = "SAVING";
+        this.currency = "USD";
+        this.balance = 0;
+    }
+
+    public void AddMoney(double newMoney) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.